src/realm/table.cpp

```diff
@@ -3694,7 +3710,7 @@ void Table::set_link(size_t col_ndx, size_t row_ndx, size_t target_row_ndx, bool
     if (col.get_weak_links())
         return;
 
-    size_t num_remaining = target_table.get_num_strong_backlinks(old_target_row_ndx);
+    size_t num_remaining = target_table.get_backlink_count(old_target_row_ndx, true);
```

Comment:
One convention to help readability is: 
```
bool only_strong_links = true;
size_t num_remaining = target_table.get_backlink_count(old_target_row_ndx, only_strong_links);
```
But that's not a blocker for me, just pointing out that we sometimes do it elsewhere in core.